tools/run_tests/run_tests.py

```diff
@@ -1048,54 +1048,82 @@ def configure(self, config, args):
         _check_compiler(self.args.compiler, ['default'])
 
     def test_specs(self):
-        return [
-            self.config.job_spec(
-                ['src/objective-c/tests/run_tests.sh'],
-                timeout_seconds=60 * 60,
-                shortname='objc-tests',
-                cpu_cost=1e6,
-                environ=_FORCE_ENVIRON_FOR_WRAPPERS),
-            self.config.job_spec(
-                ['src/objective-c/tests/run_plugin_tests.sh'],
-                timeout_seconds=60 * 60,
-                shortname='objc-plugin-tests',
-                cpu_cost=1e6,
-                environ=_FORCE_ENVIRON_FOR_WRAPPERS),
-            self.config.job_spec(
+        out = []
+        if self.config == 'dbg':
+            out += self.config.job_spec(
                 ['src/objective-c/tests/build_one_example.sh'],
                 timeout_seconds=10 * 60,
-                shortname='objc-build-example-sample',
+                shortname='ios-buildtest-example-sample',
```

Comment:
btw the name "buildtest" sounds odd.  Why not call the example build just "ios-example" (which is what it actually is).